conformance/smart-config-fhirr4-service/README.md

```diff
@@ -1 +1,134 @@
 # FHIR R4 SMART Configuration Service
+
+## Introduction
+
+This service provides API implementation of 
+[SMART Configuration API](https://www.hl7.org/fhir/smart-app-launch/#discovery-of-server-capabilities-and-configuration) 
+of a FHIR server.
+
+SMART defines a discovery document available at `.well-known/smart-configuration` relative to a FHIR Server Base URL, 
+allowing clients to learn the authorization endpoint URLs and features a server supports. This information helps clients to 
+direct authorization requests to the right endpoint, and helps clients construct an authorization request that the server 
+can support.
+
+```Supported FHIR version is 4.0.1.```
+
+### Setup and run
```

Comment:
```suggestion
## Setup and run
```

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/b1b355e6d8a01038dca36588c1af3cd618f87fa1